File: 闲来无事/从xcassets获取图片/从xcassets获取图片/ViewController.m
//
//  ViewController.m
//  从xcassets获取图片
//
//  Created by huangaengoln on 15/11/9.
//  Copyright (c) 2015年 huangaengoln. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    UIImageView *imageView=[[UIImageView alloc]initWithFrame:self.view.frame];
    imageView.image=[UIImage imageNamed:@"LaunchImage"];
    [self.view addSubview:imageView];
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 闲来无事/用字面量(literal)创建图片/用字面量(literal)创建图片/ViewController.swift
//
//  ViewController.swift
//  用字面量(literal)创建图片
//
//  Created by huangaengoln on 15/11/4.
//  Copyright © 2015年 huangaengoln. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        // 用字面量(literal)创建图片
        let image=[#Image(imageLiteral: "kitten2.png")#]
        let imageView=UIImageView(image: image)
        imageView.frame=CGRect(x: 100, y: 100, width: 100, height: 100)
        self.view.addSubview(imageView)
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

